### PR TITLE
VZ-11522: Build using golang 1.20.10 and OL8 containers

### PR DESCRIPTION
**Pull request by mgianatagh**

### CODE REVIEW

File: Dockerfile.dapper [hunk 1]
```diff
@@ -9,18 +9,11 @@ RUN yum-config-manager --enable ol7_optional_latest && \
     yum -y install bash git docker vim less file curl wget && \
     # install golang 1.17
     yum install -y oracle-golang-release-el7 && \
```
Comment (bobdonat) [inline Dockerfile.dapper:11]: is this sill needed?

File: Dockerfile.dapper [hunk 2]
```diff
@@ -9,18 +9,11 @@ RUN yum-config-manager --enable ol7_optional_latest && \
...
-    yum-config-manager --enable ol7_developer_golang117 && \
-    yum install -y golang-1.17.5 && \
+    yum-config-manager --enable ol7_developer_golang118 && \
+    yum install -y golang-1.18.5 && \
```
Comment (bobdonat) [inline Dockerfile.dapper:13]: should this be 1.20.10

File: package/Dockerfile [hunk 3]
```diff
@@ -1,24 +1,29 @@
-FROM ghcr.io/oracle/oraclelinux:7-slim
-RUN yum -y update && \
-    yum -y install openssh && \
```
Comment (bobdonat) [inline package/Dockerfile:3]: openssh no longer needed?

File: Dockerfile.dapper (hunk 1 above)
Comment (mgianatagh) [reply Dockerfile.dapper:11]: Unfortunately yes, unless I want to go through the effort of getting rid of dapper.  What really ends up happening is dapper does the build first, then I rebuild using golang 1.20.10 in the Dockerfile and copy those binaries.

File: Dockerfile.dapper (hunk 2 above)
Comment (mgianatagh) [reply Dockerfile.dapper:13]: No, the actual build take gets used is in Dockerfile.  Unfortunately there is not a golang 1.20.10 available on OL7 and not sure if there will be.  And I have to use OL7 here to install docker.

File: package/Dockerfile (hunk 3 above)
Comment (mgianatagh) [reply package/Dockerfile:3]: it is on the base image or it wasn't needed

File: Dockerfile.dapper (hunk 2 above)
Comment (mgianatagh) [reply Dockerfile.dapper:13]: I checked the image rancher/gitjob:v0.3.0 and it was not on there

File: package/Dockerfile (hunk 3 above)
Comment (mgianatagh) [reply package/Dockerfile:3]: I checked the image rancher/gitjob:v0.3.0 and it was not on there

**Review by bobdonat — APPROVED** [review]

**Pull request merged**